### DISCUSSION TITLE
0.3.0

### DISCUSSION THREAD

**Discussion by Filip-Drabinski** [Filip-Drabinski/Godot.DependencyInjection#23] [Announcements]
Released 2023-09-22

<hr /><em>This discussion was created from the release <a href='https://github.com/Filip-Drabinski/Godot.DependencyInjection/releases/tag/0.3.0'>0.3.0</a>.</em>

**Comment by delustra** [comment]
Hey, I'm trying to set up DI according to instruction. Here is what Godot gives me, when I try to launch the project:

![image](https://github.com/Filip-Drabinski/Godot.DependencyInjection/assets/6017282/638ee54a-934f-40f2-bbed-1f51e683ea16)

and that's the script I'm injecting:
![image](https://github.com/Filip-Drabinski/Godot.DependencyInjection/assets/6017282/93729da1-aad8-441e-8959-2175ab0f5332)
